go.mod

```diff
@@ -28,3 +28,6 @@ replace (
 	k8s.io/client-go => k8s.io/client-go v0.18.8
 	k8s.io/code-generator => k8s.io/code-generator v0.18.8
 )
+
+// DO NOT SUBMIT
+replace knative.dev/reconciler-test => ../reconciler-test
```

Comment:
undo this?

Reply:
I need https://github.com/knative-sandbox/reconciler-test/pull/35